scripts/globals/spells/trust/shikaree_z.lua

```diff
@@ -15,6 +23,38 @@ end
 
 spell_object.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.message_offset.SPAWN)
+
+    -- TODO: Load/Apply MODs from mob_pool_mods instead
+    mob:addMod(xi.mod.MPP, 100)
+    mob:addMod(xi.mod.STORETP, 174)
+    mob:addMod(xi.mod.JUMP_TP_BONUS, 164)
```

Comment:
Where have these numbers come from?

Reply:
Based on BG-Wiki comments, I muddled with TP Returns during testing to aim for the following values:

Melee Swing: ~205 TP Returned
Jump: ~655 TP Returned